src/pages/Autolending/DefaultRateDropdown.vue

```diff
@@ -99,12 +99,4 @@ export default {
 </script>
 
 <style lang="scss">
-@import 'settings';
```

Comment:
This chunk of CSS was repeated for all these filters. I removed it and added it in AutolendingWho. New design changes the typography of these titles to black and bold.